app/views/DR/ReportScreen/ReportScreenQuestions.js

```diff
@@ -129,7 +129,7 @@ export default function ReportScreenQuestions({ navigation }) {
             ]}
             onPress={() => {
               setDialogVisible(false);
-              navigation.goBack();
+              navigation.navigate('HomeScreen');
```

Comment:
Doing this you're pushing a new screen to the stack. You should use .popToTop() instead